scripts/dockutil

```diff
@@ -584,18 +580,14 @@ def addItem(pl, add_path, replace_label=None, position=None, before_item=None, a
     if tile_type == 'file-tile':
         new_item = {'GUID': new_guid, 'tile-data': {'file-data': {'_CFURLString': add_path, '_CFURLStringType': 0},'file-label': label_name, 'file-type': 32}, 'tile-type': tile_type}
     elif tile_type == 'directory-tile':
-        if subprocess.Popen(['/usr/bin/sw_vers', '-productVersion'],
```

Comment:
Oh no we've dropped support for Mac OS X Tiger.  🤣

Reply:
Heh yep. I figure it was worth dropping.